model/network.yaml

```diff
@@ -22,9 +22,9 @@ groups:
               value: 'unix'
               brief: "Unix domain socket"
         brief: >
-          [OSI Transport Layer](https://osi-model.com/transport-layer/) or
-          [Inter-process Communication method](https://en.wikipedia.org/wiki/Inter-process_communication).
-          The value SHOULD be normalized to lowercase.
+          [OSI transport layer](https://osi-model.com/transport-layer/) or
+          [inter-process communication method](https://en.wikipedia.org/wiki/Inter-process_communication).
```

Comment:
not a big deal, but using sentence case makes sense to me as the easiest way to stay consistent